lib/client.js

```diff
@@ -35,21 +36,23 @@ function timeoutReject (reject, name, client, paramsArray) {
         if (notAllWhitelisted) {
           throw new Error('Illegal bulk call - `instances.create` must be called separately.')
         }
-        return defaultTimer(client, reject)
+        return defaultTimer(client, reject, actions)
       }
     }
     default: {
-      return defaultTimer(client, reject)
+      return defaultTimer(client, reject, actions)
     }
   }
 }
 
-function defaultTimer (client, callback) {
+function defaultTimer (client, callback, actions) {
```

Comment:
Can we leave the callback as the last argument? It helps make the code more readable. If callback is the last argument then we can also pass in a inline anonymous function as callback and still have readable code.

Reply:
Yep good suggestion 👍